internal/container/run.go

```diff
@@ -18,6 +19,17 @@ func Run(ctx context.Context, docker client.CommonAPIClient, ctrID string, out,
 	if err := docker.ContainerStart(ctx, ctrID, types.ContainerStartOptions{}); err != nil {
 		return errors.Wrap(err, "container start")
 	}
+
+	info, err := docker.Info(ctx)
+	if err != nil {
+		return errors.Wrap(err, "getting docker info")
+	}
+
+	if info.OSType == "windows" {
+		// wait for logs to show
+		time.Sleep(time.Second)
+	}
```

Comment:
Based on our slack conversation can we create an issue to keep track of this work and add a reference to the code so we can track this workaround.

```suggestion
	// FIXME: <issue number>
	if info.OSType == "windows" {
		// wait for logs to show
		time.Sleep(time.Second)
	}
```

Reply:
No time.Sleep is needed with the subsequent changes to `container.Run` below.